{"name": "Cancel void request of new card", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id='wrapper']/div[3]/div/div/div[5]/div[2]/label\"}\n  (Session info: chrome=88.0.4324.146)\n\n", "trace": "  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\cancelvoidrequeststeps.py\", line 69, in step_impl\n    totalCalAmt = context.driver.find_element_by_xpath(\"//*[@id='wrapper']/div[3]/div/div/div[5]/div[2]/label\").text.replace('Total amount: $', '')\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 394, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given the user enter job order number in filter textbox", "status": "passed", "start": 1612853065751, "stop": 1612853079276}, {"name": "When he click on the filter result", "status": "passed", "start": 1612853079276, "stop": 1612853094153}, {"name": "And submit the void request", "status": "passed", "start": 1612853094155, "stop": 1612853094388}, {"name": "Then respose should be successful", "status": "passed", "start": 1612853094388, "stop": 1612853094558}, {"name": "And status should display as \"Pending\" in void request page", "status": "passed", "start": 1612853094558, "stop": 1612853096788}, {"name": "Given the user enter job order number in filter textbox in void request page", "status": "passed", "start": 1612853096788, "stop": 1612853115846}, {"name": "When he click the filtered result", "status": "passed", "start": 1612853115846, "stop": 1612853115973}, {"name": "And check the topup request information(top-up amount, job order number and number of cards)", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id='wrapper']/div[3]/div/div/div[5]/div[2]/label\"}\n  (Session info: chrome=88.0.4324.146)\n\n", "trace": "  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\cancelvoidrequeststeps.py\", line 69, in step_impl\n    totalCalAmt = context.driver.find_element_by_xpath(\"//*[@id='wrapper']/div[3]/div/div/div[5]/div[2]/label\").text.replace('Total amount: $', '')\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 394, in find_element_by_xpath\n    return self.find_element(by=By.XPATH, value=xpath)\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 978, in find_element\n    'value': value})['value']\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\lukegoh\\desktop\\python projects\\automationtesting\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1612853115975, "stop": 1612853126234}, {"name": "And reject request with acceptable reason", "status": "skipped", "start": 1612853126235, "stop": 1612853126235}, {"name": "Then result should display rejected submission response", "status": "skipped", "start": 1612853126235, "stop": 1612853126235}, {"name": "And the filtered result  should have status \"Cancelled\"", "status": "skipped", "start": 1612853126235, "stop": 1612853126235}], "start": 1612853065749, "stop": 1612853126235, "uuid": "329906a9-d020-494f-b7d6-e79a049480b0", "historyId": "e23d4a27feb0668a01162765c684380d", "fullName": "Cancellation of void request: Cancel void request of new card", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Cancellation of void request"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}